src/components/Button/Button.tsx

```diff
@@ -4,13 +4,14 @@ import styles from './Button.module.css';
 import { Icon } from '../Icon/Icon';
 export interface Props {
   /**
-   * Button reference
+   * Visually hidden button text (but text is still accessible to assistive technology).
+   * This overrides `text`
    */
-  buttonRef?: any;
+  'aria-label'?: string;
   /**
-   * Toggle between full width button (true) and default button width
+   * Button reference
    */
-  block?: boolean;
```

Comment:
Removed `block` but kept `fullWidth` (since it seems like they are intended to serve the same function).